main.cpp

```diff
@@ -1,157 +1,157 @@
+        for example: 
+            noun: lightBulb.
+            action: turns on.  
+                This is a bad action because someone else turns on the light.  the light doesn't turn itself on. 
+            a better action would be: 
+                illuminates room
+                This is a better action, because this is what lightbulbs do.  
+            other actions that would work:
+                action: consume electricity
+                action: burn out and destroy filament.
+
+        
+ 3) write out how you'd call that action in pseudo code, in the space after the plain-english action
+ 
+ 4) If the action requires multiple words, use camelCaseToNameIt
+     don't forget the semi-colon after each statement
+ */
+
+
+//  example)
+//  Noun:       arm                     // 1)
+//  action 1:   the arm extends         // 2)
+arm.extend();                           // 3)
+//  action 2:   the arm flexes          // 2)
+arm.flex();                             // 3)
+//  action 3:   the arm rotates conter-clockwise
+arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
+
+
+//  1)
+//  Noun: car
+//  action 1: the car's engine starts
+car.startEngine();
+//  action 2: the car accelerates
+car.accelerate();
+//  action 3: the car decelerates
+car.decelerate();
+//  2)
+//  Noun: submarine
+//  action 1: the submarine dives
+submarine.dive();
+//  action 2: the submarine surfaces
+submarine.surface();
+//  action 3: the submarine extends its periscope 
+submarine.extendPeriscope();
+//  3)
+//  Noun: multimeter
+//  action 1: the multimeter measures a dc voltage
+multimeter.measureDCVoltage();
+//  action 2: the multimeter measures an ac voltage
+multimeter.measureACVoltage();
+//  action 3: the multimeter meaures an ac frequency
+multimeter.measureACFrequency();
+//  4)
+//  Noun: tapeMachineTransport
+//  action 1: the tapeMachine's transport moves forward
+tapeMachineTransport.moveForward();
+//  action 2: the tape machines transport moves backward 
+tapeMachineTransport.moveBackward();
+//  action 3: the tape machine's transport stops
+tapeMachineTransport.stop();
+//  5)
+//  Noun: monitor
+//  action 1: the monitor selects its hdmi1 input
+monitor.selectInputHDMI1();
+//  action 2: the monitors power turns on
+monitor.powerOn();
+//  action 3: the monitors volume is turned up
+monitor.volumeUp();
+//  6)
+//  Noun: bluRayPlayer
+//  action 1: the bluRayPlayer's tray is opened
+bluRayPlayer.openTray();
+//  action 2: the bluRayPlayers menu is displayed
+bluRayPlayer.displayMenu();
+//  action 3: the bluRayPlayer advances to the next chapter
+bluRayPlayer.nextChapter();
+//  7)
+//  Noun: wire
+//  action 1: a wire connects a source to a destination
+wire.connectSourceToDestination();
+//  action 2: a wire is disconnected from a source
+wire.disconnectSource();
+//  action 3: a wire is disconnected from a destination
+wire.disconnectDestination();
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.